lib/bootloader_setup.pm

```diff
@@ -1238,4 +1240,9 @@ sub create_encrypted_part {
     assert_script_run "echo nots3cr3t | cryptsetup $luks_type luksFormat -q --force-password /dev/${disk}1";
 }
 
+sub type_boot_parameters {
+    my (@params) = @_;
+    type_string(" @params ", max_interval => get_var('TYPE_BOOT_PARAMS_FAST') ? undef : utils::VERY_SLOW_TYPING_SPEED);
```

Comment:
you can simplify getting the value with

```perl
get_var('TYPE_BOOT_PARAMS_FAST', utils::VERY_SLOW_TYPING_SPEED)
```

as the second parameter of `get_var` is what you want to supply as default value

Reply:
It is a boolean, I don't want to pass a value manually.

Reply:
then don't use `get_var` which is used to get the value

Reply:
This is how it is done in the code base for booleans.

Reply:
@dzedro, @okurz, would you be happy with?

```perl
type_string(" @params ", max_interval => check_var('TYPE_BOOT_PARAMS_FAST', 1) ? undef : utils::VERY_SLOW_TYPING_SPEED);
```

Reply:
@SergioAtSUSE If you think this ugly static way will help testers speed up their local runs, I don't mind. Good luck

Reply:
@dzedro, it would be more useful with constructive critics, in the way of "For a boolean, it should look like this" instead of just "I don't like" ;)

Reply:
I didn't use I don't like critic, but ok, message received.

Reply:
@SergioAtSUSE So this is a boolean so, let's stick with check_var. Later in the future if needed, we can revisit @dzedro's idea to use the parameter as a proxy for passing max_interval